src/components/about-dialog/about-dialog.js

```diff
@@ -0,0 +1,40 @@
+import template from "./about-dialog.html";
+import { MDCDialog } from "@material/dialog";
+
+export class AboutComponent {
+  constructor(mountPoint) {
+    this.mountPoint = mountPoint;
+  }
+
+  showDialog() {
+    this.dialog.show();
+  }
+
+  handleOk() {
+    console.log("accepted");
+  }
+
+  handleCancel() {
+    console.log("declined");
+  }
+
```

Comment:
You don't need this metod. You don't need button cancel too.

Reply:
What do you think about it?

Reply:
And I don't think you need ok handler too. I left it in my components for future actions.